deployment/ansible/raster-foundry.yml

```diff
@@ -15,6 +15,6 @@
     - name: Change directory on login
       lineinfile: dest=/home/vagrant/.bashrc regexp=^raster-foundry_PROFILE line="cd /opt/raster-foundry/"
 
-    - name: Set Environment variable for AWS profile
-      lineinfile: dest=/etc/environment regexp=^AWS_PROFILE line="AWS_PROFILE={{aws_profile}}"
```

Comment:
Seemed like a slightly better practice (we were already using `.bashrc` above. I tried just doing `source /home/vagrant/.bashrc` in provisioning to get the environment to stick during provisioning (as you guessed below) but that didn't work.